jetcd-core/src/test/java/io/etcd/jetcd/MaintenanceTest.java

```diff
@@ -92,7 +92,7 @@ public void testSnapshotChunks() throws ExecutionException, InterruptedException
         final AtomicLong count = new AtomicLong();
         final CountDownLatch latcht = new CountDownLatch(1);
 
-        maintenance.snapshot(new StreamObserver<>() {
+        maintenance.snapshot(new StreamObserver<SnapshotResponse>() {
```

Comment:
is this really needed for Java 8

Reply:
i'll double check, but looks like, that compiler can't infer this type and asks for 
`public void onNext(Object value)` otherwise